src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+html {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
Avoid using tag names for styling, with the exception of `html` and `body`. In this case, it's acceptable to style the `html` tag, but ensure it's necessary for the layout. If it is not required for the global layout, consider using a class instead.